content/.htaccess

```
@@ -3,3 +3,5 @@ RewriteEngine On
 RewriteRule "^en/(.+)$" "/$1" [R=301,DPI,L]
 RewriteRule "^docs/en/(.+)$" "/docs/$1" [R=301,DPI,L]
 RewriteRule "^docs/(zh-CN|zh-TW|ja|ko|fr)/(.+)$" "/$1/docs/$2" [R=301,DPI,L]
```

```suggestion
RewriteRule "^en/(.+)$" "/$1" [R=301,DPI]
RewriteRule "^docs/en/(.+)$" "/docs/$1" [R=301,DPI]
RewriteRule "^docs/(zh-CN|zh-TW|ja|ko|fr)/(.+)$" "/$1/docs/$2" [R=301,DPI]
```

Perhaps now we cannot set these rules with `L`?